src/seedu/addressbook/data/person/Contact.java

```diff
@@ -0,0 +1,35 @@
+package seedu.addressbook.data.person;
+/**
+ * Provides basic methods and attributes for the Email Class, Address Class and Phone Class
+ * Contains two attributes, value (String of Address, Email or Phone) and the boolean isPrivate
+ * Contains methods for toString, hashCode, getting the private attribute isPrivate and 
+ */
```

Comment:
Good job adding header comments =).
Perhaps it is too verbose? Eg. line 4 and 5?
Perhaps `Represents the contact details in the addressbook`.
As this is a more general class, it is better to describe it generally and not go into details like `... the Email Class, Address Class and Phone Class".